CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,74 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, gender identity and expression, level of experience,
+nationality, personal appearance, race, religion, or sexual identity and
+orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or
+advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic
+  address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
```

Comment:
I'd like to leave it as-is so we stay true to the original CoC. We take a pretty relaxed approach to our "professional setting" so telling jokes and being silly is fine as long as the jokes aren't demeaning or exclusive of others along the lines described. So probably fine to make the joke about VB the programming language, but not jokes that demean women.

Also, we understand that people make mistakes and have different ideas about what is considered appropriate. We're not going to hell-ban you for the first infraction unless it's way over the line.

> Project maintainers are responsible for clarifying the standards of acceptable
> behavior and are expected to take appropriate and fair corrective action in
> response to any instances of unacceptable behavior.

Consider this document the start of the conversation, not the end. For the most part, I don't see anything in here so far that goes against the spirit of this. Everyone that's participated has been pretty great. We've had a few bad eggs, but most were still somewhat respectful.